scripts/plot_docking_results.py

```diff
@@ -0,0 +1,69 @@
+import sys, os, argparse
+import numpy as np
+import plotly.express as px
+
+sys.path.append(f"{os.path.dirname(os.path.dirname(os.path.abspath(__file__)))}")
```

Comment:
Just marking this one as having the same problem as with previous scripts (as in https://github.com/choderalab/covid-moonshot-ml/pull/39/files#r978836159 )

Reply:
we do this with just about every single script I think! We should change all this in a PR

Reply:
> Yeah sounds like it could be replaced by an except IndexError: maybe?
done here: https://github.com/choderalab/covid-moonshot-ml/pull/39/commits/292253997d5da5c8ed32e0fdfc4bef46ae1cc109

Reply:
added issue: https://github.com/choderalab/covid-moonshot-ml/issues/48